modules/policy_document/variables.tf

```
@@ -1,6 +1,13 @@
 variable "template" {
   description = "Filepath to the policy document template, relative to `template_paths`"
   type        = string
+  default     = null
+}
+
+variable "templates" {
+  description = "Filepath to the policy document template, relative to `template_paths`"
+  type        = string
+
```

give this a default with an empty list...


```suggestion
  type        = string
  default     = []
```